testsuite/classlibrary/TestPatternProxy.sc

```diff
@@ -366,4 +366,19 @@ TestPatternProxy : UnitTest {
 
 
 	}
+
+	test_pbindef_arg_order {
+		var pairs;
+
+		Pbindef(\test).clear;
+		Pbindef(\test, \a, 1, \b, 2, \c, 3);
+
+		pairs = Pbindef(\test).source.pairs;
+		this.assertEquals(pairs[0,2..], [\a, \b, \c], "Pbindef key order should correspond to code");
```

Comment:
this assert and the line above are unnecessary, probably best to remove them. this test should only ensure that adding new keys inserts, not appends; any other behavior should be guaranteed by a separate test